build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -144,12 +120,30 @@ def __str__(self):
         f"cpu_features=[{features}]",
     ]
     params = ", ".join(params)
-    return f"Android device <{params}>"
+    return f"{self.platform_type.value} device <{params}>"
+
+  def get_cpu_arch_revision(self) -> str:
+    if self.cpu_abi == "arm64-v8a":
+      return self.__get_arm_cpu_arch_revision()
```

Comment:
FYI: the logic in `__get_arm_cpu_arch_revision` is not exactly correct. The subversion cannot be easily deduced just with cpuinfo features. We eventually need to drop the subversion and only report as `ARMv8-A`. Though even for armv9 architectures, Android still returns `arm64-v8a` as the CPU ABI.. Not sure there is an straightforward way to differentiate that..

Reply:
I'll keep it as it is right now. Filed an issue to track this #8631